### ISSUE TITLE
Quickstart rule testing doesn't work with non-8080 ports

### ISSUE THREAD

**Issue by Crowbrammer** [firebase/quickstart-nodejs#86]
Browser: N/A
Browser version: N/A
Operating system: Windows 
Operating system version: 8.1

What steps will reproduce the problem:
1. Create a firebase.json with a port set to 8025
```json
{
  "firestore": {
    "rules": "firestore.rules"
  },
  "emulators": {
    "firestore": {
      "port": 8080
    }
  }
}
```
2. Update the coverageUrl to be 8025 in test.js
3. Run it.

What is the expected result?
Run the tests on the rule set for the Firestore db.

What happens instead of that?
```js
  0 passing (1s)
  1 failing

  1) "before all" hook:
     Error: 14 UNAVAILABLE: Trying to connect an http1.x server
      at Object.exports.createStatusError (node_modules\grpc\src\common.js:91:15)
      at Object.onReceiveStatus (node_modules\grpc\src\client_interceptors.js:1204:28)
      at InterceptingListener._callNext (node_modules\grpc\src\client_interceptors.js:568:42)
      at InterceptingListener.onReceiveStatus (node_modules\grpc\src\client_interceptors.js:618:8)
      at callback (node_modules\grpc\src\client_interceptors.js:845:24)
```

**Comment by ryanpbrewster** [comment]
You can use environment variables to configure the testing SDK
```
export FIRESTORE_EMULATOR_HOST=localhost:8025
```

**Comment by samtstern** [comment]
Fixed by #85 which makes the tests look at the JSON file.
